evm/src/test/java/org/hyperledger/besu/evm/gascalculator/OsakaGasCalculatorTest.java

```diff
@@ -25,7 +25,11 @@ class OsakaGasCalculatorTest {
   @Test
   void testPrecompileSize() {
     OsakaGasCalculator subject = new OsakaGasCalculator();
-    assertThat(subject.isPrecompile(Address.precompiled(0x14))).isFalse();
+    // past last l1 precompile address
+    assertThat(subject.isPrecompile(Address.precompiled(0x12))).isFalse();
+    // past last l2 precompile address
+    assertThat(subject.isPrecompile(Address.precompiled(0x0101))).isFalse();
+    assertThat(subject.isPrecompile(Address.P256_VERIFY)).isTrue();
```

Comment:
Please add:
`assertThat(subject.isPrecompile(Address.precompiled(0x00))).isFalse();`

Currently failing for Osaka